quickspot.js

```diff
@@ -1238,6 +1252,19 @@
 		};
 
 		/**
+		 * Sort full dataset
+		 * Provides a simple alphabetical sort that should be used when no search has been made.
+		 * While this can provide some odd results for numeric key_values, it is the default for
```

Comment:
How so? assuming the keys were numbers, i'd assume it should work as expected?

have tested with;
```
[{"__keyvalue":1},{"__keyvalue":11},{"__keyvalue":21},{"__keyvalue":22}].sort(function(a, b){ return (a.__keyvalue > b.__keyvalue) ? 1 : -1; });
```

Reply:
Values are converted to string, so they're sorted lexicographically. Would've been fine for the original use case (i.e. where names are the key value), so it's likely never come up.

```
const dataset = [
    {id: 2, item: 'apple', colour: 'red'},
    {id: 8, item: 'banana', colour: 'yellow'},
    {id: 0, item: 'peach', colour: 'orange'},
    {id: 1, item: 'orange', colour: 'orange'},
    {id: 22, item: 'orange', colour: 'orange'},
    {id: 33, item: 'orange', colour: 'orange'},
    {id: 11, item: 'orange', colour: 'orange'}
];

const controlStore = quickspot.datastore({
    data: dataset,
    key_value: 'id'
});

controlStore.store.all();
// controlStore.store.results === [{id: 0, ...}, {id: 1, ...}, {id: 11, ...}, {id: 2, ...}, {id: 22, ...}, {id: 33, ...}, {id: 8, ...}]
```

Reply:
Ah, yea, forgot about that :/